src/Service/main.cpp

```diff
@@ -95,5 +95,13 @@ int main(int argc, char** argv) {
 
   exit_requested = false;
   orbit_service::OrbitService service{grpc_port, dev_mode};
-  return service.Run(&exit_requested);
+  auto result = service.Run(&exit_requested);
+
+  if (!result.has_error()) return 0;
+
+  std::puts(result.error().message().c_str());
+  std::ignore = std::fflush(stdout);
+
+  constexpr int kExitCodeIndicatingErrorMessage = 42;
```

Comment:
I'm torn if I should put this constant somewhere where it can be accessed both from the client and the service. But I don't know where and if it's worth to create a separate module for it. What are your thoughts on this?

Reply:
That's what I want to avoid - adding a header which serves as a dumping ground for all the things that don't fit anywhere else.

I could add a header just for service exit codes though.

Reply:
> That's what I want to avoid - adding a header which serves as a dumping ground for all the things that don't fit anywhere else.

Totally agree with this, let's avoid that.

> I could add a header just for service exit codes though.

You mean to `OrbitBase`? I have a question: personally I see "base" modules as something that is independent from the project itself. Am I wrong here? We already break the rule with `ThreadConstants.h` and `Profiling.h`, but indeed I don't like that very much.

This said I'm also a bit torn, because of course this constant is defined in multiple places, but I also don't like adding a module for a single constant. I would say that if you only have one exit code it's completely fine as it is, if we end up with three or more we should reconsider. 
Maybe you can add a comment on the two definitions pointing to each other and saying they should be kept in sync (although those comments easily get obsolete if code gets moved around, so I'm not even 100 % sure this is good).

Reply:
Yes. You exactly summarized my thinking. I guess I will leave it as-is. I will think about adding a comment though. Thx!